features/bootstrap/SmokeContext.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+use Behat\Behat\Tester\Exception\PendingException;
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+
+/**
+ * Defines application features from the specific context.
+ */
+class SmokeContext implements Context, SnippetAcceptingContext
+{
+    private $fs;
+
+    private $lastOutput;
+
+    /**
+     * Initializes context.
+     *
+     * Every scenario gets its own context instance.
+     * You can also pass arbitrary arguments to the
+     * context constructor through behat.yml.
+     */
+    public function __construct()
+    {
+        $this->fs = new \Symfony\Component\Filesystem\Filesystem();
+    }
+
+    /**
+     * @Given I have a Docker Compose file that contains one container
+     */
+    public function iHaveADockerComposeFileThatContainsOneContainer()
+    {
+        $path = __DIR__ . '/../../../tmp/';
+        $this->fs->remove($path);
+        $this->fs->mkdir($path);
+        chdir($path);
+
+        $config = <<<EOF
+memcached:
+  image: memcached
+EOF;
+
+        $this->fs->dumpFile('docker-compose.yml', $config);
+    }
+
+    /**
+     * @Given this container is running
+     */
+    public function thisContainerIsRunning()
+    {
+        shell_exec(__DIR__ . '/../../bin/dock-cli up');
```

Comment:
Might be good to move this `__DIR__ . '/../../bin/dock-cli up'` used in many places at least in a private method.